vpr/src/route/connection_box_lookahead_map.cpp

```diff
@@ -65,10 +91,12 @@ class CostMap {
         }
     }
 
+    // Map node id to segment id.
     int node_to_segment(int from_node_ind) {
         return segment_map_[from_node_ind];
     }
 
+    // Find the cost entry for a given segment to connection box, and distance from segment to connection box.
```

Comment:
s/distance/offset

Reply:
No it is distance in this case?  It is not an offset